tests/suites/1_stateful/05_formats/05_01_compact/05_01_02_load_compact_copy_max_size.sh

```diff
@@ -33,6 +33,6 @@ echo "---row_count"
 echo "select count(*) from t1" | $MYSQL_CLIENT_CONNECT
 
 echo "---block_count"
-echo "select block_count from fuse_snapshot('default','t1')" | $MYSQL_CLIENT_CONNECT
+#echo "select block_count from fuse_snapshot('default','t1')" | $MYSQL_CLIENT_CONNECT
```

Comment:
cc @youngsofun 

This pr just modifies the max_threads default value, before this pr the value of max_threads is 16. Now it will be determined automatically.

And the result of block_count is not equal 4.

Reply:
you can use clickhouse http handler to post the query with `max_threads` params.

Reply:
> you can use clickhouse http handler to post the query with `max_threads` params.

Good idea. Let's try.

Reply:
https://github.com/datafuselabs/databend/actions/runs/3425559377/jobs/5706546925

05_01_02_load_compact_copy_max_size:                                    [ FAIL ] - result differs with:
--- /runner/_work/databend/databend/tests/suites/1_stateful/05_formats/05_01_compact/05_01_02_load_compact_copy_max_size.result	2022-11-09 05:35:17.022386923 +0000
+++ /runner/_work/databend/databend/tests/suites/1_stateful/05_formats/05_01_compact/05_01_02_load_compact_copy_max_size.stdout	2022-11-09 05:39:26.770978465 +0000
@@ -3,4 +3,4 @@
 ---row_count
 1000
 ---block_count
-4
+3

Looks like this modify is not useful.